data/database.py

```diff
@@ -1398,7 +1398,7 @@ class BlobUpload(BaseModel):
     repository = ForeignKeyField(Repository)
     uuid = CharField(index=True, unique=True)
     byte_count = BigIntegerField(default=0)
-    sha_state = ResumableSHA256Field(null=True, default=resumablehashlib.sha256)
+    sha_state = ResumableSHA256Field(null=True, default=rehash.sha256)
```

Comment:
Are we 100% certain this is backwards compatible? Please add a TODO and JIRA to verify

Reply:
https://issues.redhat.com/browse/PROJQUAY-771